src/_nebari/stages/infrastructure/template/aws/modules/network/main.tf

```diff
@@ -3,7 +3,6 @@ resource "aws_vpc" "main" {
 
   enable_dns_support   = true
   enable_dns_hostnames = true
-  enable_classiclink   = false
```

Comment:
Note: This was removed since it's no longer supported as of August 23, 2023 - https://aws.amazon.com/blogs/aws/ec2-classic-is-retiring-heres-how-to-prepare/